transcom/mymove
# Display PPM Origin and Destination address

## B-18329

## Summary
The pickup and destination address should show on the MTO tab as a TOO.
The pickup address is required to be entered by the customer but if there is no destination location the destination duty location should be shown.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create PPM move and submit as SC. Take note of pickup and destination address.
2. As TOO find move and click "Move task order" tab to confirm addresses

## Screenshots
<img width="1034" alt="Screenshot 2024-03-13 at 8 33 00 AM" src="https://github.com/transcom/mymove/assets/146969726/3558c462-999f-498a-b3ea-f9fa4b8e73ca">
<img width="1013" alt="Screenshot 2024-03-13 at 8 29 16 AM" src="https://github.com/transcom/mymove/assets/146969726/bb8194a6-89ad-4f20-947e-e22972d37196">
